backend/portal/models.py

```diff
@@ -48,17 +50,54 @@ class Content(models.Model):
     admin_comment = models.CharField(max_length=255, null=True, blank=True)
```

Comment:
In line 50, created_date should probably be immutable. 
created_date = models.DateTimeField(default=timezone.now, editable=False)

Update save to
def save(self, *args, **kwargs):
    if not self.pk:
        self.created_date = timezone.now()  # Set created_date to the current time
    else:
        if 'update_fields' in kwargs and 'created_date' in kwargs['update_fields']:
            kwargs['update_fields'].remove('created_date')

    prev = self.__class__.objects.filter(id=self.id).first()
    super().save(*args, **kwargs)

    if prev is None:
        self._on_create()
        return

    if self.status != prev.status:
        self._on_status_change()

Reply:
If you look in the serializer for content, you'll see that created_date is read only so it is enforced there. 
Technically an admin could change the created date, but there is no way to prevent this since you can always change a row in a db. In other words you can't enforce atomicity at the db level